pkg/cache/manager.go

```diff
@@ -136,3 +86,25 @@ func (m *Manager) CacheUsage(ctx context.Context, blobID string) (snapshots.Usag
 
 	return usage, nil
 }
+
+func (m *Manager) RemoveBlobCache(blobID string) error {
+	blobCachePath := path.Join(m.cacheDir, blobID)
+	blobCacheSuffixedPath := path.Join(m.cacheDir, blobID+dataFileSuffix)
+	blobChunkMap := path.Join(m.cacheDir, blobID+chunkMapFileSuffix)
+	blobMeta := path.Join(m.cacheDir, blobID+metaFileSuffix)
+
+	// NOTE: Delete chunk bitmap file before data blob
+	stuffs := []string{blobChunkMap, blobMeta, blobCachePath, blobCacheSuffixedPath}
+
+	for _, f := range stuffs {
+		err := os.Remove(f)
+		if err != nil {
+			if errors.Is(err, os.ErrNotExist) {
+				log.L.Warnf("file %s doest not exist.", f)
```

Comment:
For the new nydusd version, the `blobCachePath` always not exists, perhaps we just need to skip the error.

Reply:
fixed